app/models/eligibilities/v3/determinations/individual_market_determination.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
```

Comment:
CMS calls this the "ExchangeEligibility" worth discussing whether we match them or go with individual